set-archive-status and delete-data subcommand do not work on OpenSearch serverless Registry

### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

The following commands fail with 404 error. The lidvid requested is not found when using an OpenSearch serverless Registry.

```
./bin/registry-manager set-archive-status \
    -auth /Users/loubrieu/Documents/pds/registry/es-auth.txt \
    -es file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml \
    -lidvid urn:nasa:pds:insight_rad:data_derived::7.0 \
    -status archived
```

```
./bin/registry-manager delete-data \
    -auth /Users/loubrieu/Documents/pds/registry/es-auth.txt \
    -es file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml \
    -lidvid urn:nasa:pds:insight_rad:data_raw:hp3_rad_raw_00478_20200401_121608::1.0
```


### 🕵️ Expected behavior

I expected the commands to succceed.

### 📜 To Reproduce

Follow temporary manual https://docs.google.com/document/d/19FVATXpfbKxmoGjcq9Zxpuml4pX-nBZRlNzleGLttPA/edit


### 🖥 Environment Info

_No response_

### 📚 Version of Software Used

latest SNAPSHOT versions:
registry-common 1.6.0-SNAPSHOT
registry-mgr: 4.7.0-SNAPSHOT

as merged on main.


### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

It sounds like the error comes from the special character encoding management in the AWS SDK used to query the OpenSearch database.

### 🎉 Integration & Test

_No response_

Hi @al-niessner ,

The `set-archive-status` command work but the `delete-data` fails.

My mvn dependency confirms I am using the version 3.0.03-SNAPSHOT of openseach-java:

  `  [INFO] |  +- org.opensearch.client:opensearch-java:jar:3.0.0-SNAPSHOT:compile`

The command I am using on the MCP dev opensearch is, with the logs :

```
% ./bin/registry-manager delete-data \                  
    -auth /Users/loubrieu/Documents/pds/registry/es-auth.txt \
    -es file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml \
    -lidvid urn:nasa:pds:insight_rad:data_raw:hp3_rad_raw_00478_20200401_121608::1.0
Elasticsearch URL: file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml
null

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[ERROR] Request failed: [http_exception] server returned 404
```

Did you test that yourself ?

Thanks

--- a/src/main/java/gov/nasa/pds/registry/mgr/cmd/data/DeleteDataCmd.java
+++ b/src/main/java/gov/nasa/pds/registry/mgr/cmd/data/DeleteDataCmd.java
@@ -85,15 +85,6 @@ public class DeleteDataCmd implements CliCommand
             return;
         }
         
-        id = cmdLine.getOptionValue("lid");
-        if(id != null)
-        {
-            this.filterMessage = "              LID: " + id;
-            regQuery.createFilterQuery("lid", id);
-            refsQuery.createFilterQuery("collection_lid", id);
-            return;
-        }
-
         id = cmdLine.getOptionValue("packageId");
         if(id != null)
         {
@@ -104,16 +95,7 @@ public class DeleteDataCmd implements CliCommand
             return;
         }
 
-        if(cmdLine.hasOption("all"))
-        {
-            this.filterMessage = "Delete all documents ";
-            regQuery.createMatchAllQuery();
-            refsQuery.createMatchAllQuery();
-            
-            return;
-        }
-
-        throw new Exception("One of the following options is required: -lidvid, -lid, -packageId, -all");
+        throw new Exception("One of the following options is required: -lidvid, -packageId");
     }
     
     

